source/renderer/app/stores/StakingStore.js

```diff
@@ -40,18 +56,11 @@ export default class StakingStore extends Store {
   }
 
   @computed get stakePools(): Array<StakePool> {
-    // return this.stakePoolsRequest.result ? this.stakePoolsRequest.result : [];
-    return STAKE_POOLS;
+    return this.stakePoolsRequest.result ? this.stakePoolsRequest.result : [];
   }
 
   @computed get delegatingStakePools(): Array<StakePool> {
```

Comment:
Do we need this @computed value?

Reply:
@tomothespian I think this is usable for the Delegation wizard - we can use it to show the "Recently used stake pools" list. We just need to use this list, order stake pools by rank and pick the 6 best ranked to show.

Reply:
@daniloprates after discussion with @nikolaglumac about this one, the conclusion is that we don’t need this one because I have another method for “Recently used stake pools”!

Reply:
@tomothespian it's also a good way of listing a value, as MobX helps to reduce unnecessary renderings.

Reply:
@tomothespian this is the default way we return API fetched data, like [addresses](https://github.com/input-output-hk/daedalus/blob/v2-integration/source/renderer/app/stores/AddressesStore.js#L15), [wallets](https://github.com/input-output-hk/daedalus/blob/v2-integration/source/renderer/app/stores/WalletsStore.js#L506), [transactions](https://github.com/input-output-hk/daedalus/blob/v2-integration/source/renderer/app/stores/TransactionsStore.js#L94). How do you suggest the Staking store returns the fetched pools?

Reply:
@tomothespian feel free to remove it :) better to do with the code you already have.

Reply:
@daniloprates deal!